site/_posts/2017-04-17-Fine-tuning-Fine.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Last Week in ManageIQ: And Finally . . . Fine-tuning Fine"
```

Comment:
I'd prefer if the "Last Week in ManageIQ is left in, to be consistent with [most of the posts](http://manageiq.org/blog/tags/LWIMIQ/) so far. Only bad example is last week's post :sweat_smile: